lib/ship_fosdick/shipments_updater.rb

```diff
@@ -1,11 +1,12 @@
 module ShipFosdick
```

Comment:
Much better name here. Really it's updating a multitude of shipments so the plural makes way more sense.